p11-kit/delete-object.c

```diff
@@ -70,7 +70,7 @@ delete_object (const char *token_str)
 		goto cleanup;
 	}
 
-	if (p11_kit_uri_parse (token_str, P11_KIT_URI_FOR_TOKEN, uri) != P11_KIT_URI_OK) {
+	if (p11_kit_uri_parse (token_str, P11_KIT_URI_FOR_ANY, uri) != P11_KIT_URI_OK) {
```

Comment:
Why not `P11_KIT_URI_FOR_OBJECT_ON_TOKEN`? ANY also matches tokens/modules which is not very useful in this context.

Reply:
I thought that it might be good to allow deleting all objects across multiple tokens etc. I can change that.

Reply:
That is what `P11_KIT_URI_FOR_OBJECT_ON_TOKEN` means. PKCS#11 can also be used to match against tokens or modules, which are not objects, and thus irrelevant for these commands.